reviews/sandbox/2022-wasmedge-annual.md

```diff
@@ -0,0 +1,155 @@
+# WasmEdge Runtime-2022-Annual Review
+
+## Background
+
+WasmEdge is a lightweight, high-performance, and extensible WebAssembly runtime. The ultimate goal for WasmEdge is to bring cloud-native and serverless application paradigms to edge, streaming, web3, and SaaS computing environments.
+
+The unique features and advantages of WasmEdge include the following.
+
+* One of the [fastest](https://github.com/WasmEdge/WasmEdge#performance) WebAssembly VMs on the market (based on **LLVM AOT**)
+* WasmEdge feature extensions
+  * **Network sockets** ([Rust](https://github.com/second-state/wasmedge_wasi_socket), C and [JavaScript](https://github.com/second-state/wasmedge-quickjs#http-request) SDKs)
+  * **Async polling** (for Rust Future and JS async)
+  * Tensorflow, Pytorch, OpenVINO inference ([Tutorial](https://wasmedge.org/docs/category/ai-inference))
+  * Key value storage
+  * Database connector including MySQL-based databases, Redis and PostgreSQL. ([Tutorial](https://wasmedge.org/docs/category/database-drivers))
+  * **Gas meters** for resource constraints
+* JavaScript support
+  * [ES6 module](https://github.com/second-state/wasmedge-quickjs#es6-module-support) and std API support
+  * **Implement JS APIs in Rust** ([Tutorial](https://www.secondstate.io/articles/embed-rust-in-javascript/))
+  * Import C native shared library functions as JS functions ([Tutorial](https://www.secondstate.io/articles/call-native-functions-from-javascript/))
+* Cloud native management and orchestration
+  * [CRI-O and Kubernetes compatibility](https://wasmedge.org/docs/category/deploy-wasmedge-apps-in-kubernetes)
+  * Docker Desktop integration
+  * Sidecar apps in Kubernetes-based service meshes
+  * Dapr microservices with Dapr WasmEdge SDK ([Tutorial](https://github.com/second-state/dapr-sdk-wasmedge))
+* Cross-platform support
+  * Linux OSes dated back to 2010 for both x86 and ARM CPUs
+  * Mac OS X for both x86 and m1 CPUs
+  * Windows
+  * Microkernel and RTOS (e.g., the highly secure [seL4 microkernel](https://github.com/second-state/wasmedge-sel4))
+  * OpenWRT
+  * RISC-V
+  * Android
+* Easy extensibility
+  * Extend and enhance the runtime with custom functions in [C](https://wasmedge.org/docs/embed/c/reference/0.12.0#host-functions) or [GO](https://wasmedge.org/docs/embed/go/reference/0.12.0#host-functions))
+* Easy to embed into a host application
+  * Embed WasmEdge functions in [C](https://wasmedge.org/docs/category/c-sdk-for-embedding-wasm-functions), [Go](https://wasmedge.org/docs/category/go-sdk-for-embedding-wasm-functions), and [Rust](https://wasmedge.org/docs/category/rust-sdk-for-embedding-wasm-functionst)-based host applications.
```

Comment:
Since the WASMEdge Doc's website is going through a complete revamp a number of links mentioned are not working. @juntao @alabulei1